app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.tsx

```diff
@@ -11,11 +11,13 @@ import AnalyticsContext from '../context/analytics';
 import { InPersonContext } from '../context';
 import UploadContext from '../context/upload';
 import { LOCATIONS_URL } from './in-person-location-post-office-search-step';
+import PostOfficeNoResults from '../../address-search/components/post-office-no-results';
```

Comment:
The linter modifies this to `import PostOfficeNoResults from '@18f/identity-address-search/components/post-office-no-results'`

I may need to first create and publish PostOfficeNotResults component. Then I can come back in and use 18f.